package.json

```diff
@@ -1,11 +1,8 @@
 {
-  "name": "sentry-docs-develop",
```

Comment:
is this change intentional?

Reply:
Yeah this wasn't intentional. I ran yarn init, instead of yarn run init. Reverted this change.